src/asm/rgbasm.1

```diff
@@ -7,89 +7,187 @@
+.Op Fl E
 .Op Fl g Ar chars
+.Op Fl h
 .Op Fl i Ar path
+.Op Fl L
 .Op Fl M Ar dependfile
 .Op Fl o Ar outfile
 .Op Fl p Ar pad_value
 .Op Fl r Ar recursion_depth
-.Ar file
+.Op Fl V
+.Op Fl v
+.Op Fl w
+.Ar
 .Sh DESCRIPTION
 The
 .Nm
-program creates an object file from an assembly source file.
+program creates a RGB object file from an assembly source file.
 The input
 .Ar file
 can be a file path, or
 .Cm \-
 denoting
 .Cm stdin .
-Its arguments are as follows:
+.Pp
+Note that options can be abbreviated as long as the abbreviation is unambiguous:
+.Fl -verb
+is
+.Fl -verbose ,
+but
+.Fl -ver
+could also be
+.Fl -version .
+The arguments are as follows:
 .Bl -tag -width Ds
-.It Fl b Ar chars
+.It Fl b , Fl -binary-digits Ar chars
 Change the two characters used for binary constants.
 The defaults are 01.
-.It Fl D Ar name Ns Op = Ns Ar value
+.It Fl D , Fl -define Ar name Ns Oo = Ns Ar value Oc
 Add string symbol to the compiled source code. This is equivalent to
 .Ar name
 .Cm EQUS
 .Qq Ar "value"
 in code. If a value is not specified, a value of 1 is given.
-.It Fl E
+.It Fl E , Fl -export-all
 Export all labels, including unreferenced and local labels.
-.It Fl g Ar chars
-Change the four characters used for binary constants.
+.It Fl g , Fl -gfx-chars Ar chars
+Change the four characters used for gfx constants.
 The defaults are 0123.
-.It Fl h
+.It Fl h , Fl -halt-without-nop
 By default,
 .Nm
 inserts a
-.Sq nop
+.Cm nop
 instruction immediately after any
-.Sq halt
+.Cm halt
 instruction.
 The
 .Fl h
 option disables this behavior.
-.It Fl i Ar path
+.It Fl i , Fl -include Ar path
 Add an include path.
-.It Fl L
+.It Fl L , Fl -preserve-ld
 Disable the optimization that turns loads of the form
 .Sy LD [$FF00+n8],A
 into the opcode
 .Sy LDH [$FF00+n8],A
 in order to have full control of the result in the final ROM.
-.It Fl M Ar dependfile
+.It Fl M , Fl -dependfile Ar dependfile
 Print
 .Xr make 1
 dependencies to
 .Ar dependfile .
-.It Fl o Ar outfile
+.It Fl o , Fl -output Ar outfile
 Write an object file to the given filename.
-.It Fl p Ar pad_value
+.It Fl p , Fl -pad-value Ar pad_value
 When padding an image, pad with this value.
 The default is 0x00.
-.It Fl r Ar recursion_depth
+.It Fl r , Fl -recursion-depth Ar recursion_depth
 Specifies the recursion depth at which RGBASM will assume being in an infinite loop.
-.It Fl V
+.It Fl V , Fl -version
 Print the version of the program and exit.
-.It Fl v
+.It Fl v , Fl -verbose
 Be verbose.
+.It Fl W , Fl -warning Ar warning
+Set warning flag
+.Ar warning .
+A warning message will be printed if
+.Ar warning
+is an unknown warning. See the
+.Sx DIAGNOSTICS
+section for a list of warnings.
 .It Fl w
-Disable warning output.
+Disable all warning output, even when turned into errors.
+.El
+.Sh DIAGNOSTICS
+Warnings are diagnostic messages that indicate possibly erroneous behavior that
+does not necessarily compromise the assembling process. The following options
```

Comment:
New sentence goes on a new line. All versions of `troff` use this to know where to put correct inter‐sentence spacing (visible as wider space between sentences in PDF output, and two spaces in the terminal).

Reply:
Didn't know about that! I was trying to stick to the 80-char-line limit, but if that is the case, then this should be prioritized. Will fix.